pkgs/by-name/gu/guile-hoot/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoreconfHook,
+  guile,
+  pkg-config,
+  texinfo,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-hoot";
+  version = "0.5.0";
+
+  src = fetchFromGitLab {
+    owner = "spritely";
+    repo = "guile-hoot";
+    rev = "v${version}";
+    hash = "sha256-n8u0xK2qDLGySxiYWH882/tkL8ggu3hivHn3qdDO9eI=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    texinfo
+  ];
+  buildInputs = [
+    guile
+  ];
```

```suggestion
  nativeBuildInputs = [
    autoreconfHook
    guile
    pkg-config
    texinfo
  ];
  buildInputs = [
    guile
  ];
  strictDeps = true;
```
guile is used both as library and compiler. #178468